armi/bookkeeping/mainInterface.py

```diff
@@ -80,30 +80,49 @@ def _activateDB(self):
             # load case before going forward with normal cycle
             runLog.important("MainInterface loading from DB")
 
-            # Load the database from the point just before start cycle and start node
-            # as the run will continue at the begining of start cycle and start node,
-            # and the database contains the values from the run at the end of the
-            # interface stack, which are what the start start cycle and start node
-            # should begin with.
+            # Load the database from the point just before start cycle and start node as the run
+            # will continue at the begining of start cycle and start node, and the database contains
+            # the values from the run at the end of the interface stack, which are what the start
+            # start cycle and start node should begin with.
 
-            # NOTE: this should be the responsibility of the database, but cannot
-            # because the Database is last in the stack and the MainInterface is
-            # first
+            # NOTE: this should be the responsibility of the database, but cannot because the
+            # database is last in the stack and the MainInterface is first
             dbi.prepRestartRun()
             self.r.p.cycle = self.cs["startCycle"]
             self.r.p.timeNode = self.cs["startNode"]
 
     def _moveFiles(self):
-        # check for orificed flow bounds files. These will often be named based on the
-        # case that this one is dependent upon, but not always. For example, testSassys
-        # is dependent on the safety case but requires physics bounds files.  now copy
-        # the files over
+        """
+        At the start of each run, arbitrary lists of user-defined files can be copied around.
+
+        This logic is controlled by the settings ``CONF_COPY_FILES_FROM`` & ``CONF_COPY_FILES_TO``.
+
+        ``CONF_COPY_FILES_FROM`` :
+
+        - List of files to copy (cannot be directories).
+        - Can be of length zero (that just means no files will be copied).
+        - The file names listed can use the ``*`` glob syntax, to reference multiple files.
+
+
+        ``CONF_COPY_FILES_TO`` :
+
+        - List of directories to copy the files into.
+        - Can be of length zero; all files will be copied to the local dir.
+        - Can be of length one; all files will be copied to that dir.
+        - The only other valid length for this list _must_ be the same length as the "from" list.
```

Comment:
This is all really useful context for a user. Unfortunately users won't typically see this info because it is in the docstring of a private method. Is there a better place to put this that will make it user-facing?

Reply:
Hmm... I can' think of where else it would belong.